app/javascript/components/track/dig-deeper-components/community-videos/CommunityVideo.tsx

```diff
@@ -68,7 +68,7 @@ export function CommunityVideosFooter({
   )
   return (
     <footer className="text-p-small text-textColor6">
-      {t('communityVideoFooter.wantYourVideoFeatured')}
+      {t('communityVideoFooter.wantYourVideoFeatured')}&nbsp;
```

Comment:
Do we not normally do `{" "}` in these cases?

Reply:
yeah, {" "} is probably better here.